modules/integration/src/test/scala/scala/cli/integration/RunScriptTestDefinitions.scala

```diff
@@ -573,6 +573,21 @@ trait RunScriptTestDefinitions { _: RunTestDefinitions =>
       }
     }
 
+  test("script file with shebang header and no extension run with scala-cli") {
+    val msg    = "compiled and ran"
+    val inputs = TestInputs(
+      os.rel / "scriptWithShebang" ->
+        s"""|#!/usr/bin/env -S ${TestUtil.cli.mkString(" ")} shebang -S 2.13
```

Comment:
```suggestion
        s"""|#!/usr/bin/env -S ${TestUtil.cli.mkString(" ")} shebang
```
Why override the Scala version here?